spec/features/user_login_logout_functionality_spec.rb

```diff
@@ -19,21 +13,11 @@
       expect(current_path).to eq(dashboard_path)
       click_link("Logout")
       click_link("Login")
-      fill_in("Email", with: "Jeffy")
+      fill_in("Email", with: "J@gmail.com")
       fill_in("Password", with: 'pass')
+      click_button 'Login'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.